tests/integration/high_availability/high_availability_helpers.py

```diff
@@ -228,6 +239,45 @@ async def relate_mysql_and_application(
     )
 
 
+def deploy_chaos_mesh(ops_test: OpsTest, namespace: str) -> None:
+    """Deploy chaos mesh to the provided namespace.
+
+    Args:
+        ops_test: The ops test framework
+        namespace: The namespace to deploy chaos mesh to
+    """
+    env = os.environ
+    env["KUBECONFIG"] = os.path.expanduser("~/.kube/config")
+
+    subprocess.check_output(
+        " ".join(
+            [
+                "tests/integration/high_availability/scripts/deploy_chaos_mesh.sh",
+                namespace,
+            ]
+        ),
+        shell=True,
+        env=env,
+    )
+
+
+def destroy_chaos_mesh(ops_test: OpsTest, namespace: str) -> None:
+    """Deploy chaos mesh to the provided namespace.
```

Comment:
```suggestion
    """Remove chaos mesh from the provided namespace.
```

Reply:
Good eye. Addressed in be15a67bd659c981a1705d287ef1d10a9e930f38